src/watermelonSidebar.ts

```diff
@@ -114,12 +114,40 @@ export default class WatermelonSidebar implements vscode.WebviewViewProvider {
           });
           break;
         }
-        default:{
-          this.sendMessage({
-            command: "",
-          });
-        }
+        case "docs": {
+          // Send Event to VSC Telemtry Library
+          this.reporter.sendTelemetryEvent("getDocs");
+
+          //get current filepath with vs code
+          let filePath = vscode.window.activeTextEditor?.document.uri.fsPath as string;
+          let mdFilePath = filePath + ".md";
+
+          //define fileystem as fs
+          let fs = require("fs");
+
+          // check if the markdown file exists
+          if (fs.existsSync(mdFilePath)) {
+            // if it does, open it in the editor
+            let mdFile = await vscode.workspace.openTextDocument(mdFilePath);
 
+            // open md file on a split view
+            vscode.window.showTextDocument(mdFile, {
+              viewColumn: vscode.ViewColumn.Beside
+            });
+          } else {
+            // show error message if the markdown file does not exist
+            vscode.window.showErrorMessage(
```

Comment:
Why not use the [error file](https://github.com/watermelontools/wm-extension/blob/dev/src/utils/vscode/showErrors.ts)?